README.md

```diff
@@ -1,25 +1,35 @@
-<!-- ![](https://img.shields.io/badge/Microverse-blueviolet) ![](https://img.shields.io/badge/JavaScript-yellow)
+![](https://img.shields.io/badge/Microverse-blueviolet) ![](https://img.shields.io/badge/JavaScript-yellow)
 
-# WEATHER APP
+# TO DO LIST
 
-> A weather forecast site using the [OpenWeather](https://openweathermap.org/) API.
-
-Original project specification: [The Odin Project](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript/lessons/weather-app).
+This project features:
 
+- Use webpack to bundle JavaScript.
+- Learn how to use proper ES6 syntax.
+- Use ES6 modules to write modular JavaScript.
+- Set up a new project with webpack that is based on the webpack exercise you have already completed.
+- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). - The index.html file must be set as a template using the HTML Webpack Plugin.
+- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
+1.description [string].
+2.completed [bool].
+3.index: [number].
+- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
+- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should apear in order of the index values for each task.
+- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
 
 ## Built With
 
 - Html, CSS, and JavaScript;
 - Git and GitHub.
 
-## Screenshots
+<!-- ## Screenshots
 
 ![screenshot](./assets/imgs/screenshot2.png)
 ![screenshot](./assets/imgs/screenshot3.png)
-![screenshot](./assets/imgs/screenshot4.png)
+![screenshot](./assets/imgs/screenshot4.png) -->
 
 ## Live Demo
-[Weather App](https://anapdh.github.io/weather-app/)
+[Weather App](https://anapdh.github.io/todo-app/dist.index.html)
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to change the title of the Live demo link form `Weather App` to something like `To Do List`